gdx/src/com/badlogic/gdx/graphics/g2d/BitmapFont.java

```diff
@@ -817,6 +863,26 @@ public boolean hasGlyph (char ch) {
 		 * {@link #getGlyphs(GlyphRun, CharSequence, int, int, Glyph)} should be be used to shape a string of characters into a list
 		 * of glyphs. */
 		public Glyph getGlyph (char ch) {
+			if (Character.isHighSurrogate(ch)) {
+				highSurrogate = ch;
+				return null;
+			} else if (Character.isLowSurrogate(ch)) {
+				if (highSurrogate == null) {
+					assert false : "No surrogate found for 0x" + Integer.toHexString(ch);
+					return null;
+				}
+				final Glyph[] page = surrogates == null ? null : surrogates[highSurrogate - HIGH_SURROGATE_MIN];
+				highSurrogate = null;
+				if (page == null) {
+					// allow people who choose to run with asserts enabled get notified when characters don't work
```

Comment:
No need for this comment.

Reply:
Assertions aren't used much in libGDX. The `Gdx.app` logging might be more appropriate?